x-pack/plugins/infra/public/pages/logs/page_toolbar.tsx

```diff
@@ -99,6 +107,7 @@ export const LogsToolbar = injectI18n(({ intl }) => {
                 jumpToTime={jumpToTargetPositionTime}
                 startLiveStreaming={startLiveStreaming}
                 stopLiveStreaming={stopLiveStreaming}
+                setSurroundingLogsId={setSurroundingLogsId}
```

Comment:
(in reference to the earlier comments)

This is where we could instead just pass a modified `startLiveStreaming` prop instead.